test/functional/feature_assumeutxo.py

```diff
@@ -419,10 +444,10 @@ def check_tx_counts(final: bool) -> None:
                 self.wait_until(lambda: n.getindexinfo() == completed_idx_state)
 
 
-        # Node 2: all indexes + reindex
-        # -----------------------------
+        # Node 2: all indexes + reindex + divergent chain
```

Comment:
I think this edit here (and in the log below) isn't needed. "all indexes + reindex" is describing the configuration of the node. Divergent chain is just one of the test cases that this node runs through. I think just having the logs in the test you already have is enough.

Reply:
Fixed